task/blue-green-deploy/0.1/clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
Not a blocker: .. just a thought 

The catalog TEP doesn't mention where the supporting resources need to be placed. I think it is okay to keep all resources required to run the task in the task's definition directory itself, so that a tool can `kubectl apply -f <path/to/task/version>` 

Perhaps we should discuss and come to a consensus 
cc: @vdemeester @afrittoli @bobcatfish @chmouel

Reply:
Let's add this topic for discussion for today's meeting!

Reply:
as discussed in the WG call and with Sunil, we shall add all the supporting resources to `support/` dir along with the task definition and will also be documented in the TEP.